lib/fs.js

```diff
@@ -74,6 +78,7 @@ module.exports = {
             });
           }
         );
+        compiler.hasWebpackDevMiddlewareAssetEmittedCallback = true;
```

Comment:
We don't actually need to create a flag for the `compiler`. e.g.
```
let hasWebpackDevMiddlewareAssetEmitted = false;
compiler.hooks.emit.tap('WebpackDevMiddleware', (compilation) => {
// ...

hasWebpackDevMiddlewareAssetEmitted = true;

```

Reply:
@unbug developers can add new compiler in runtime